_posts/2022-02-28-dont-lgtm-code-reviews.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: Doesn't look good to me — a requiem for thorough code reviews
+date: 2022-02-28
+tags: code cloudnative consultants deployment continuous-delivery devops patterns review software test testing
+author: Simon Bracegirdle
+image: img/dont-lgtm-code-reviews.jpg
+description:
+  Do you give a cursory review and "LGMT!" on pull requests? Read this to find out how you could bring more rigour to your code review practices and improve the software you deliver.
```

Comment:
do you give a cursory glance and hastily slap the trusty "LGMT!" comment on pull requests?

Reply:
I feel you could add some spice here